woocommerce/wc-admin
# Theme color: Use dynamic variable from user preferences

Partially addresses https://github.com/woocommerce/woocommerce-admin/issues/4544

Colors for buttons and various other core Gutenberg elements should get wp-admin theme colors.

### Detailed test instructions:

1. Poke around the app, see button colors rendered correctly.
2. Edit your user profile
<img width="853" alt="Screen Shot 2020-06-12 at 9 31 29 AM" src="https://user-images.githubusercontent.com/1922453/84443986-d1031980-ac94-11ea-8362-1b26c73a6938.png">
3. Change to a new color scheme.
<img width="859" alt="Screen Shot 2020-06-12 at 9 33 18 AM" src="https://user-images.githubusercontent.com/1922453/84444006-da8c8180-ac94-11ea-9f7f-c32fea7d5fa2.png">
4. See the new colors reflected.
<img width="434" alt="Screen Shot 2020-06-12 at 9 48 18 AM" src="https://user-images.githubusercontent.com/1922453/84444025-e4ae8000-ac94-11ea-8931-711d9b2b8598.png">

### Note

There are still some instances of colors being overriden where they shouldn't be (breadcrumbs?), some colors need to be tweaked (date range picker), and areas where overrides need to be placed (OBW). I'd like to follow up with those fixes, this PR is just the mechanism for getting G2 colors shown.

Instead of darken, lighten I used `color` function. See https://github.com/postcss/postcss-color-function#notes-for-former-sass-users

### Changelog Note:

- Enhancement: Use Core colors for styling accents.
